adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -104,6 +105,9 @@ public void run() {
                     validateAcquireTokenRequest(authRequest);
                     performAcquireTokenRequest(callbackHandle, activity, useDialog, authRequest);
                 } catch (final AuthenticationException authenticationException) {
+                    mAPIEvent.setSuccessStatus(false);
```

Comment:
>setSuccessStatus [](start = 30, length = 16)

can this be renamed? setSuccessStatus sounds like setting the success case status.